developer-docs-site/docs/reference/move.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Move on Aptos"
+slug: "move"
+sidebar_position: 0
+---
+
+# Move on Aptos
+
+The Aptos blockchain consists of validator nodes that run a consensus protocol that agrees upon the ordering of transactions and their output when executed on the MoveVM. Each validator node translates transactions along with the current blockchain ledger state as input into the VM, which processes this input to produce a changeset or storage delta as output. Once consensus agrees and commits to the output, it becomes publicly visible. In this guide, we will introduce you to core Move concepts and how they apply to developing on Aptos.
+
+## What is Move?
+
+Move is a safe and secure programming language for Web3 that emphasizes scarcity and access control. An assets in Move can be represented by or stored within *resource*. **Scarcity** is enforced by default, as unless otherwise made explicit objects and instances within Move are not copyable. Only those objects that have explicitly been defined at the bytecode layer as *copy* can be duplicated.
+
+Access control comes from both the notion of accounts as well as module access privileges. A module in Move may either be a library or a program that can be executed to create, store, or transfer assets. Move ensures that only public interfaces within a module may be accessed by other modules. In addition, structs or objects within Move can only be constructed within the module where it is defined unless the developer has explicitly defined a public constructor. Similarly, fields within a struct can only be accessed and mutated within the module that defines the struct or via public accessors and setters.
```

Comment:
I would bold **Access control** to go with your theme above for **scarcity**.

Drop `as well as`, it's wordy.

Access control comes from both the notion of accounts and module access privileges.

Drop `that can be executed`

A module in Move may either be a library or a program to create store or transfer assets.

Move ensures that only public module interfaces may be accessed by other modules.

Unless a struct or object has a public constructor, it can only be constructed within its containing module.

Similarly, fields within a struct can only be accessed within its containing module unless public accessors and setters are defined.